docs/getting-started/using-gnokey.md

```diff
@@ -0,0 +1,777 @@
+---
+id: using-gnokey
+---
+
+# Using `gnokey`
+
+## Overview
+In this tutorial, you will learn how to use the `gnokey` binary to interact with
+a Gno.land chain. You will learn how to create state-changing calls, run readonly
+queries without using gas, as well as create, sign, and broadcast airgapped
```

Comment:
What does "airgapped" mean here?

Reply:
Airgap means having a split in the transaction creation &singing and the broadcasting. I've linked the [wiki page](https://en.wikipedia.org/wiki/Air_gap_(networking)) to it so it can give a bit more context; although it is a well known term in Web3 imho

Reply:
[`dec53e7` (#2226)](https://github.com/gnolang/gno/pull/2226/commits/dec53e71e9fa90601bfc6b57721b4ca00f080631)